media/js/firefox/firstrun/firstrun.js

```diff
@@ -6,8 +6,7 @@
     'use strict';
 
     var animateSunrise = function () {
-        // Bug 1381051 sign in should redirect to about:home instead of about:newtab.
-        var redirectDest = 'about:home';
+        var redirectDest = 'about:newtab';
```

Comment:
This particular JS file is used for `/firefox/56.0/firstrun/` and below. Just checking we're ok to redirect to `about:newtab` in older versions, as well as >= 57?

Reply:
Actually, no we don't want to change it for 56 below. about:newtab is still Tiles and about:home is the old about:home (which focuses search box). So I'll undo the changes to this file, and it then won't require the bundle change either.